package.json

```diff
@@ -16,7 +16,7 @@
     "@types/node": "^18.11.9",
     "@typescript-eslint/eslint-plugin": "^6.20.0",
     "@typescript-eslint/parser": "^6.20.0",
-    "ava": "^5.3.0",
+    "ava": "^5.3.1",
```

Comment:
is bumping ava necessary to make this work? or just hygiene?

Reply:
hygiene. When I did `yarn add -D ava` it changed yarn.lock, which prompted me to bring them all up to date. I tend to put hygiene commits first in the list.

Reply:
OK.

I guess I should look at the release notes. By way of audit trail:

> What's Changed
> - Update t.like() to support Symbol keys and ignore non-enumerable properties by @gibson042 in https://github.com/avajs/ava/pull/3209
> - Fix circular selector detection in t.like() by @novemberborn in https://github.com/avajs/ava/pull/3212

-- https://github.com/avajs/ava/releases/tag/v5.3.1

Reply:
> I guess I should look at the release notes

I don't expect a reviewer to review the release notes for a patch or even minor release.

Reply:
Separated to https://github.com/Agoric/agoric-sdk/pull/9006